source/includes/_quickstart-wallets.md

```diff
@@ -1,5 +1,9 @@
 ## <em>Quickstart Wallets 🚧</em>
 
+<aside class="notice">
+Please note, wallets are not available by default. Please speak to the Goji integrations team if you wish to use this functionality.
```

Comment:
nit (Grammer Police): "Please note" should be followed by a colon rather than a comma.